src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps2_mqtt_client_module.c

```diff
@@ -37,6 +37,18 @@ typedef enum ADPS_REGISTER_STATE_TAG
     ADPS_REGISTER_STATE_REGISTERED = 4 /**< Device registration is successful. */
 } ADPS_REGISTER_STATE;
 
+enum ADPS_ERROR
+{
+    ADPS_ERROR_NONE = 0,
```

Comment:
Move out to an adps_errors.h, or move these to result.h via the result json and use ADUC_Result so that doxygen doc can be auto-generated for errors?